docs/get-started/getting-started.mdx

```diff
@@ -27,6 +27,89 @@ ds library style sheet in your entry point, in this example we use React (App.js
 
 <WindowEditor language="javascript" header={{ show: true, title: 'App.js' }} codeString="import 'dd360-ds/dd360.css'" />
 
+<br />
+
+## <span name="floating-nav">Provider</span>
+For DD360-ds to work correctly, you need to set up the ThemeProvider at the root of your application.
+
+Go to the root of your application and do this:
```

Comment:
I suggest we put an example of how it would look like in Nextjs and React 

```suggestion
Go to the root of your application and do it.

For Nextjs projects, you can do it as':
```